pkg/oauth/server/osinserver/defaults.go

```diff
@@ -25,6 +25,7 @@ func NewDefaultServerConfig() *osin.ServerConfig {
 	config.AllowGetAccessRequest = true
 	config.RedirectUriSeparator = ","
 	config.ErrorStatusCode = http.StatusBadRequest
+	config.RequirePKCEForPublicClients = true // TODO determine if we want to do this
```

Comment:
What's the case for disabling PKCE? Can we always try for PKCE against both supporting and non-supporting servers?